Dockerfile

```diff
@@ -33,3 +33,6 @@ RUN vca-install-package \
   bzip2 \
   gzip \
   xz-utils
+
+ADD entrypoint.sh /opt/entrypoint.sh
+ENTRYPOINT ["/bin/sh", "/opt/entrypoint.sh"]
```

Comment:
Although I see know that there's already a script for adding a key to `known_hosts` we just need another for doing the same buy to the `authorized_keys` file.